config/install/taxonomy.settings.yml

```diff
@@ -0,0 +1,5 @@
+maintain_index_table: true
```

Comment:
TIL you can turn off the index table. 

> * Taxonomy uses default field storage to store canonical relationships
> * between terms and fieldable entities. However its most common use case
> * requires listing all content associated with a term or group of terms
> * sorted by creation date. To avoid slow queries due to joining across
> * multiple node and field tables with various conditions and order by criteria,
> * we maintain a denormalized table with all relationships between terms,
> * published nodes and common sort criteria such as status, sticky and created.

Reply:
well thats interesting. nice find